src/main/scala/com/wavesplatform/state2/diffs/BlockDiffer.scala

```diff
@@ -38,7 +38,7 @@ object BlockDiffer extends ScorexLogging {
         })
     }
     lazy val feeDiff = Monoid[Diff].combineAll(accountPortfolioFeesMap.map { case (acc, p) =>
-      Diff(portfolios = Map(acc -> p))
+      new Diff(Map.empty, Map(acc -> p), Map.empty, Map.empty, Map.empty, Map.empty, Seq.empty)
```

Comment:
..and it does exist! Decided not to create more so noone gets confused with too specific overloadings.